docs/references/v2.openapi.json

```diff
@@ -1362,6 +1362,14 @@
               "type": "string"
             },
             "description": "ID of the room"
+          },
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "name": "query",
+            "description": "Query to filter threads. Ex: `metadata[\"status\"]:\"open\" AND metadata[\"resolved\"]:true AND metadata[\"priority\"]:1`"
```

Comment:
Should we include a link to some documentation where people can read more about the query language syntax and the possible operators?

Reply:
That's a very good point, I am wondering where that doc should live? For now I could just put a link to the node doc. But we do need a dedicated page 🤔

Reply:
If there's no place for some information, put it in a [guide](https://github.com/liveblocks/liveblocks/tree/main/guides/pages) and link to it! Going to write a quick one now.